src/manifold/include/manifold.h

```diff
@@ -118,7 +118,7 @@ class Manifold {
    */
   ///@{
   MeshRelation GetMeshRelation() const;
-  std::vector<int> GetMeshIDs() const;
+  int OriginalID() const;
```

Comment:
Here's the first big API change. Again, I think I'm going in the right direction here because it simplifies our tests. If they want to know the total set, they can scan triBary as easily as we can.